pkg/quarterdeck/db/models/orgs.go

```diff
@@ -23,12 +24,17 @@ type Organization struct {
 }
 
 // OrganizationUser is a model representing a many-to-many mapping between users and
-// organizations. This model is primarily used by the User and Organization models and
-// is not intended for direct use generally.
+// organizations and describes the role each user has in their organization. This model
+// is primarily used by the User and Organization models and is not intended for direct
+// use generally.
+//
+// NOTE: a user can only have one role in an organization, so roles must be defined as
+// overlapping sets rather than as disjoint sets where users have multiple roles.
 type OrganizationUser struct {
```

Comment:
Is this model currently used?

Reply:
No, not directly, I don't believe except for tests right now.